tests/test_pluginmanager.py

```diff
@@ -3,14 +3,12 @@
 import pytest
 
 from porcupine import pluginloader
-from porcupine.plugins.pluginmanager import show_dialog
+from porcupine.plugins.pluginmanager import create_dialog
 
 
 @pytest.fixture
 def dialog_content(mocker):
-    mocker.patch("tkinter.Toplevel.wait_window", autospec=True)
-    content = show_dialog()
-    tkinter.Toplevel.wait_window.assert_called_once()
+    content = create_dialog()
     yield content
     tkinter.Toplevel.wait_window.call_args[0][0].destroy()
```

Comment:
With my other suggestion, you can now access the dialog much more easily:

```suggestion
    dialog.destroy()
```